core/interfaces/i_ast_node_location_svg.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview The interface for an AST node location SVG.
+ * @author samelh@google.com (Sam El-Husseini)
+ */
+
+'use strict';
+
+goog.provide('Blockly.IASTNodeLocationSvg');
+
+goog.require('Blockly.IASTNodeLocation');
+
+
```

Comment:
N.B.: multiple blank lines [allowed but not encouraged](https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace).  (Here and a few other places.)

Reply:
We use 2 newlines after our requires in all of our files. I would rather address this separately in a different pass.

Reply:
Oh, OK: I think that's a great convention.  It just stood out in this PR because you don't add two blank lines between before the body of the module when the module has no `require`s, so it looked a little inconsistent.